### ISSUE TITLE
User list for image classifiers 

### ISSUE THREAD

**Issue by genkimaps** [genkimaps/wildcam#26]
Create user list for image classifiers in management section (currently only has principle investigator). Could be either by project or global user list (across projects).  **Add staff member not currently working**

**Comment by mitch-fen** [comment]
Would be good to have a list of "Contributors" on the project home page to assist in tracking and crediting down the road

**Comment by mitch-fen** [comment]
Correct. Could likely just be an open text box, or alternatively a list populated by all users assigned permissions (the first is likely easiest and most robust)
